src/scripts/components/VariableMessage.jsx

```diff
@@ -142,16 +141,20 @@ class VariableMessage extends React.Component {
     // Get flat-level list of all variables
     const variables = this.getVariables(this.props.variables);
     // Split `initialValue` to target variables
-    const split = initialValue.split(/({\w+})/g);
+    const split = initialValue.split(/({.*?})/);
+
+    const lowercaseSplit = split.map(e => e.toLowerCase());
     const available = [];
     // Loop through variables
+    console.log(lowercaseSplit);
     variables.forEach((value) => {
       const { variable } = value;
-      const isVariablePresent = split.includes(variable);
-
+      const isVariablePresent = lowercaseSplit.includes(variable.toLowerCase());
+      console.log(variable.toLowerCase(), isVariablePresent);
       // See if we've found one in our `initialValue`
       if (isVariablePresent) {
-        const variableIdx = split.indexOf(variable);
+        // If so, transform the variable into HTML
+        const variableIdx = lowercaseSplit.indexOf(variable.toLowerCase());
```

Comment:
Consider not abbreviating `Index`. I see it may have been used previously, but try to convert them or change them as you go.